staff/joel-molero/mailchimp/index.html

```diff
@@ -0,0 +1,131 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Mailchimp</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <header>
+        <div class="topbar">
+            <div class="mailchimplogo">
+                <p><img src="../mailchimp/images/img/main_logo.png" alt=""></p>
+            </div>
+            <div class="button">
+                <button class="signupfree">Sign Up Free</button>
```

Comment:
To have an adaptive button:

    font-size: 1.2vw;
    display: inline;
    white-space: nowrap;